app/views/tariffs/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<br>
+<br>
+<br>
+<h2>Редагування тарифу</h2>
+<%= form_for :tariff, url: tariffs_path do |f| %>
```

Comment:
@romigos it is duplicated part. Please move to partial _form.html.erb and use it in edit.html.erb and new.html.erb

Reply:
fixed